Resources/doc/reference/installation.rst

```diff
@@ -1,124 +1,152 @@
+.. index::
+    single: Installation
+    single: Configuration
+
 Installation
 ============
 
-* Add SonataTimelineBundle to your vendor/bundles dir via composer
+The easiest way to install ``SonataTimelineBundle`` is to require it with Composer:
 
-.. code-block:: json
+.. code-block:: bash
 
-    //composer.json
-    "require": {
-    //...
-        "sonata-project/timeline-bundle": "~2.2@dev",
-    //...
-    }
+    $ php composer.phar require sonata-project/timeline-bundle
+
+Alternatively, you could add a dependency into your `composer.json` file directly.
```

Comment:
i have to lern the difference, for me it isn't clear yet when to use one or two ticks :smile: